Userland/Libraries/LibCore/CMakeLists.txt

```diff
@@ -6,14 +6,25 @@ set(SOURCES
     ArgsParser.cpp
     Directory.cpp
     DirectoryEntry.cpp
-    DirIterator.cpp
     Environment.cpp
     File.cpp
     StandardPaths.cpp
     System.cpp
     Version.cpp
 )
 
+if (WIN32)
+    list(APPEND SOURCES
+        DirIteratorWindows.cpp
+    )
+else()
+    list(APPEND SOURCES
+        DirIterator.cpp
+    )
+endif()
```

Comment:
if there's only one source file, it's a bit easier on the eyes to do `list(APPEND SOURCES DirIteratorWindows.cpp)` in one line instead of 3.